block/submit.go

```diff
@@ -217,10 +187,15 @@ func (m *Manager) CreateNextBatchToSubmit(startHeight uint64, endHeightInclusive
 			// Remove the last block and commit from the batch
 			batch.Blocks = batch.Blocks[:len(batch.Blocks)-1]
 			batch.Commits = batch.Commits[:len(batch.Commits)-1]
+
+			if height == startHeight {
+				return nil, fmt.Errorf("batch size exceeds max size: %d", totalSize)
```

Comment:
in this case should probably say "block size exceeds max batch size" as its only one block.

Reply:
done